src/server/container_utils.h

```diff
@@ -69,6 +72,32 @@ bool IterateSet(const PrimeValue& pv, const IterateFunc& func);
 bool IterateSortedSet(robj* zobj, const IterateSortedFunc& func, int32_t start = 0,
                       int32_t end = -1, bool reverse = false, bool use_score = false);
 
+static inline StringMap* GetStringMap(const PrimeValue& pv, const DbContext& db_context) {
```

Comment:
`static inline` is a bit of an abomination :) If it's in a header I'd remove the `static`.

```suggestion
inline StringMap* GetStringMap(const PrimeValue& pv, const DbContext& db_context) {
```